charts/kubewarden-controller/Chart.yaml

```diff
@@ -35,14 +35,14 @@ annotations:
   catalog.cattle.io/display-name: Kubewarden # Only for Charts with custom UI
   catalog.cattle.io/os: linux # this means linux only, other choice here is "windows". For charts that support both, don't add this annotation
   # optional ones:
-  catalog.cattle.io/auto-install: kubewarden-crds=1.4.6
+  catalog.cattle.io/auto-install: kubewarden-crds=1.5.0-rc1
```

Comment:
We've changed the CRDs, hence I think it's fine to do a minor bump. This version LGMT